Source/WebCore/dom/ViewTransition.cpp

```diff
@@ -484,6 +485,22 @@ static ExceptionOr<void> forEachRendererInPaintOrder(NOESCAPE const std::functio
     return { };
 };
 
+static bool rendererIsFragmented(const RenderLayerModelObject& renderer)
+{
+    if (renderer.hasContinuationChainNode() || renderer.isContinuation())
+        return true;
```

Comment:
This is a very different concept than fragmentation. I am curious as to why it is included here.

Reply:
The [css-break-4 spec for 'fragmentation'](https://drafts.csswg.org/css-break-4/#fragmentation-model) has "A box can be broken into multiple fragments also due to ... higher-level [display type](https://drafts.csswg.org/css-display-4/#display-type) box splitting, e.g. [block-in-inline splitting](https://www.w3.org/TR/CSS2/visuren.html#img-anon-block) (see [CSS2§9.2](https://www.w3.org/TR/CSS2/visuren.html#box-gen))"

That was the intent of this check, and fixes the provided testcase in bugzilla.

Reply:
Could you link to the spec in the code, please? It's is not obvious why continuation would show up here in a function called fragmentation (especially when continuation is really just implementation details and it may go away soon)